tinygrad/codegen/search.py

```diff
@@ -38,7 +38,8 @@ def opt(x):
   if not opts: return "BASELINE"
   search_space = prod([len(x.choices) for x in opts])
   st = time.perf_counter()
-  optimizer = ng.optimizers.NGOpt(parametrization=ng.p.Tuple(*opts), budget=min(search_space, 200))
+  budget = int(getenv("BUDGET", 200))
```

Comment:
You don't need the int here, it'll take the dtype from the default.

Reply:
oh you are right - updated. wino PR had the int too and I got the wrong impression